### PR TITLE
samples:  updates samples to v1 (3 of 8)

### PR DESCRIPTION
**Pull request by telpirion**
This updates the java-aiplatform samples to use the v1 version of the samples.

### CODE REVIEW

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineCustomJobSample.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Google LLC
+ * Copyright 2021 Google LLC
```
Comment (averikitsch) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineCustomJobSample.java:2]: nit: no need to update

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageClassificationSample.java [hunk 2]
```diff
@@ -204,25 +200,6 @@ static void createTrainingPipelineImageClassificationSample(
...
-      ExplanationMetadata explanationMetadata = explanationSpec.getMetadata();
-      System.out.println("Metadata");
-      System.out.format("Inputs: %s\n", explanationMetadata.getInputsMap());
-      System.out.format("Outputs: %s\n", explanationMetadata.getOutputsMap());
-      System.out.format(
-          "Feature Attributions Schema_uri: %s\n",
-          explanationMetadata.getFeatureAttributionsSchemaUri());
-
       Status status = trainingPipelineResponse.getError();
       System.out.println("Error");
```
Comment (averikitsch) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageClassificationSample.java:204]: Is this suppose to error or does this need a catch stateement?

Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageClassificationSample.java:204]: No, the service might run into an error during model training that is unrelated to the request sent by the customer. There isn't an error that we can catch for this. (Same as other samples.)

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineCustomJobSample.java (hunk 1 above)
Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineCustomJobSample.java:2]: Okay, thanks for letting me know!

**Review by averikitsch — APPROVED** [review]

**Comment by telpirion** [comment]
Thank you!

**Pull request merged**